src/gui/FileRevealer.cpp

```diff
@@ -47,22 +47,19 @@ const QString& FileRevealer::getDefaultFileManager()
 #else
 	QProcess process;
 
+	QString desktopEnv = qgetenv("XDG_SESSION_DESKTOP");
+	if (desktopEnv.isEmpty())
+	{
+		desktopEnv = qgetenv("XDG_CURRENT_DESKTOP").trimmed().toLower().split(':').first();
+	}
```

Comment:
Unless you're aware of instances where `XDG_SESSION_DESKTOP` is better, please trust `XDG_CURRENT_DESKTOP`, which has the word `gnome` in it and will provide the match we need for `gio`.  Also, please do not split and trim this value as it will remove `gnome` from `ubuntu:GNOME`.

```suggestion
	QString desktopEnv = qgetenv("XDG_CURRENT_DESKTOP").trimmed().toLower();
```